### DISCUSSION TITLE
Sitemap indexing

### DISCUSSION THREAD

**Discussion by simkinsws** [patrickdemooij9/SeoToolkit.Umbraco#180] [Q&A]
Hey, i'm using you Package for umbraco i would like to know if there is a Sitemap indexing built-in ?
if im going to www.mysite.com/sitemap.xml i see ofcourse the whole sitemap enteries.

but Sitemap indexing should be like: www.mysite.com/services/sitemap.xml and now it will display only the urls under the /services/
does something like that exists in this package? or there is a way to achieve it? thanks .

**Comment by patrickdemooij9** [comment]
HI @simkinsws,
The package does support sitemaps build it. Are you asking if the package supports dedicated sitemaps for other parts of the website like products, services, etc?
